runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -131,12 +171,12 @@ class LayoutFFITypeHelpers
 	}
 
 	/**
-	 * @brief obtain the ffi_type from the layout symbol (the preceding letter of the layout type. e.g. I for INT)
+	 * @brief Obtain the ffi_type from the layout symbol (the preceding letter of the layout type. e.g. I for INT).
 	 *
-	 * @param layoutSymb[in] The layout symbol describing the type of the layout
+	 * @param layoutSymb the layout symbol describing the type of the layout
 	 * @return The pointer to the ffi_type corresponding to the layout symbol
 	 */
-	static VMINLINE ffi_type*
+	static VMINLINE ffi_type *
 	getPrimitiveFFIType(char layoutSymb)
 	{
 		ffi_type* typeFFI = NULL;
```

Comment:
Space before, not after `*`, please. See also line 229.

Reply:
Fixed everywhere.